hw/dv/sv/test_vectors/test_vectors.core

```diff
@@ -11,6 +11,7 @@ filesets:
     files:
       - test_vectors_pkg.sv
       - vectors/hmac/HMAC_RFC4868.rsp: {is_include_file: true}
+      - vectors/hmac/HMAC.rsp: {is_include_file: true}
```

Comment:
I don't know so much in security yet, but don't you think that here we should precise which kind of HMAC we are talking about in the name itself (maybe add a suffix with SHA-2)? Maybe it's enough, I just want to double check with you

Reply:
It was one file earlier with all test vectors for all digest sizes, but I decided to split it down now into 3 files, so now they are conveniently suffixed :-)